src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,47 @@
 import React from 'react';
-
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+type Props = {
+  title: string;
+  description: string;
+  imgUrl: string;
+  imdbUrl: string;
+};
```

Comment:
You can simplify your life)
```suggestion
type Props = {
  movie: Movie,
};
```

Reply:
Yeah, use your type `Movie` here, just don't forget to Exclude `imdbId`